src/command.ts

```diff
@@ -217,7 +216,16 @@ export class Command {
               ])
             );
           }
-          process.exit();
+          process.exitCode = 0;
+          const timeout = setTimeout(
+            (cmd: any) => {
```

Comment:
Think this needs to be an async function as well & await track, so that the track actually happens